### DISCUSSION TITLE
Deleting conversations on mobile

### DISCUSSION THREAD

**Discussion by jcdiv47** [danny-avila/LibreChat#2390] [Q&A] [answered]
I have been enjoying LibreChat so much lately that I also started using it on mobile(basically Safari on iPhone). However, I found the interaction a tiny little bit annoying when trying to delete a conversation. What happened basically was I pop out the convo list/menu, select(focus on) a conversation, then the list is folded, and I am brought into the conversation page, then I have to click the triangle (now with focus on the conversation title) to open the menu again, only then does the delete button is shown.

Nothing is breaking but I am just wondering if anyone also notices it and if there is an alternative solution to the interaction. Please let me know if I explain my question clearly.

**Comment by danny-avila** [comment, marked as answer]
Glad you're enjoying it! I use it on mobile on the go, too. I understand, I think chatgpt had a similar issue but they've improved this. It's been on my backlog to do the same
